daemon/src/main/scala/org/apache/gearpump/cluster/main/AppSubmitter.scala

```diff
@@ -38,7 +38,7 @@ object AppSubmitter extends App with ArgumentsParser {
     "namePrefix" -> CLIOption[String]("<application name prefix>", required = false, defaultValue = Some("")),
     "jar" -> CLIOption("<application>.jar", required = true))
 
-  def start : Unit = {
+  def main(akkaConf: Config, args: Array[String]): Unit = {
```

Comment:
Is it possible to return the application id from `ClientContext` here ? Then, we can report info like `Successfully submitted application 1 wordcount` back to users.